### PR TITLE
feat(event): Add moderators event review process.

### PR DESCRIPTION
**Pull request by jainmickey**
Done

### CODE REVIEW

**Comment by theskumar** [comment]
@jainmickey any further updates here?

**Comment by jainmickey** [comment]
@theskumar basic functionality is done.

File: fossevents/events/services.py [hunk 1]
```diff
@@ -51,6 +59,27 @@ def send_event_update_mail(event, moderators_email):
                extra_context, 'email/event_update_creator_email.html')
 
 
+def send_event_review_mail(review, moderator):
+
+    # get all moderator's email
+    moderators_email = User.objects.filter(Q(is_staff=True) | Q(is_moderator=True) |
+                                           Q(is_superuser=True)).distinct().values_list('email', flat=True)
```
Comment (toanant) [inline fossevents/events/services.py]: Filtering on `is_moderator` will fulfill the use-case here, `is_staff` & `is_superuser` not needed.

File: fossevents/events/services.py [hunk 2]
```diff
@@ -59,10 +88,42 @@ def send_confirmation_mail(event, is_update=False):
     :param is_update boolean
     """
 
-    # get all end users both candidates and employers
-    moderators_email = User.objects.filter(is_staff=True).values_list('email', flat=True)
+    # get all moderator's email
+    moderators_email = User.objects.filter(Q(is_staff=True) | Q(is_moderator=True) |
+                                           Q(is_superuser=True)).distinct().values_list('email', flat=True)
```
Comment (toanant) [inline fossevents/events/services.py]: same goes here, can be added as separate function to fetch moderators email to re-use at other places.

File: fossevents/events/services.py (hunk 1 above)
Comment (jainmickey) [reply fossevents/events/services.py]: Done

File: fossevents/events/services.py (hunk 2 above)
Comment (jainmickey) [reply fossevents/events/services.py]: @toanant created a user manager to get filtered queries

**Review by satyaakam — APPROVED** [review]

**Pull request merged**